backends/LxcBackend.py

```diff
@@ -252,21 +256,22 @@ def display(self, user):
         cdisplay = ":" + str(self.id + 2)
         hostdisplay = os.getenv("DISPLAY")
         os.system("xhost local: >/dev/null 2>&1")
-        command="DISPLAY=" + hostdisplay + " Xephyr -title \"Xephyr " + self.name + "\" -br -ac -dpms -s 0 -resizeable " + cdisplay + " 2>/dev/null & \
+        command = "DISPLAY=" + hostdisplay + " Xephyr -title \"Xephyr " + self.name + "\" -br -ac -dpms -s 0 -resizeable " + cdisplay + " 2>/dev/null & \
             export DISPLAY=" + cdisplay + " ; \
             while ! setxkbmap -query 1>/dev/null 2>/dev/null ; do sleep 0.1s ; done ; \
             xfce4-session 2>/dev/null & \
             while ! pidof xfce4-panel 1>/dev/null ; do sleep 0.1s ; done ; \
             setxkbmap -display " + hostdisplay + " -print | xkbcomp - " + cdisplay + " 2>/dev/null"
-            #xkbcomp " + str(hostdisplay) + " :" + str(displaynum)
```

Comment:
comments with xkbcomp to https://unix.stack should be indented as setxkbmap

Reply:
Aligning all these comments, generates a warning so I will put an empty line between these and the rest.